### ISSUE TITLE
TOTK research: Weapon Durability unknown values

### ISSUE THREAD

**Issue by KrisCris** [marcrobledo/savegame-editors#282]
I added two Master Sword (glitched) into my inventory and set their durability to 1. However, only one of them is displayed as "almost broken" in game.

So I was wondering if this has something to do with the durability calculation, like if those weapons w/ fused materials calculate the durability differently? or if there are some hidden values not displayed on the save editor...

Game save attached.
[progress.zip](https://github.com/marcrobledo/savegame-editors/files/11598844/progress.zip)

**Comment by marcrobledo** [comment]
I'm pretty sure there are other values for fused weapons, but I haven't had time to take a look at it.

**Comment by ApacheThunder** [comment]
The Master Sword uses a -1 value I think for indicate it's proper durablity value. It shows up in the editor as a really large number. You can't set it your self though as the input field is limited to a int values and not uint. Might need to resolve that if you want to set the correct durablity stat on unbreakable version of Master Sword. ;)

**Comment by Truinto** [comment]
The reason this doesn't work is that weapons store a second 'fuse' durability. It's unlocked while the weapon is fused and consumed first (while unlocked, if you unfuse it eats the base durability again). The editor does not show this second durability at all. The restore durability button also ignores this value.

Side note; this second durability depends solely on the base weapon, not the material. For most weapons it's 25. There is probably a third durability counter for when you fuse two weapons (this is a guess).

**Comment by KrisCris** [comment]
> There is probably a third durability counter for when you fuse two weapons (this is a guess).

I once saw a video testing fused durability saying that the durability of both weapons decreases at the same time, and the one whose durability reaches 0 earlier breaks first. (No guarantee this is true)

**Comment by marcrobledo** [comment]
AFAIK, maximum durability allowed (without Durability Up modifier set, of course) is:
- not fused=base weapon durability
- fused with material=base weapon durability+25
- fused with another weapon=base weapon durability+fused weapon durability
Or that's at least how I calculate it in the editor.

Also, this is not exactly related but...
I managed to find and fix a bug with the durability input fields in the editor, which were not getting their maximum value allowed updated when changing an item. So now if you change your sword, bow or shield and/or its fused item, the maximum durability value will be fixed.

My bad, I thought I fixed that one already some weeks ago :-(
